inabox/templates/testconfig-anvil-docker.yaml

```diff
@@ -45,7 +45,6 @@ services:
       SRS_LOAD: 2900
       CHALLENGE_ORDER: 3000
       LOG_LEVEL: "debug"
-      LOG_FORMAT: "text"
```

Comment:
do we not want this?

Reply:
The default is set to "json", so I was going with that. Is there a reason we want text logging?